test/support/integration/plugins/module_utils/docker/common.py

```diff
@@ -334,11 +334,13 @@ def __init__(self, argument_spec=None, supports_check_mode=False, mutually_exclu
 
         if not HAS_DOCKER_PY:
             if NEEDS_DOCKER_PY2:
-                msg = missing_required_lib("Docker SDK for Python: docker")
-                msg = msg + ", for example via `pip install docker`. The error was: %s"
```

Comment:
Changing this file makes no sense, this is a copy that is only used in ansible/ansible's integration tests. You should modify https://github.com/ansible-collections/community.docker/blob/main/plugins/module_utils/common.py#L264 instead.

Reply:
Jinx 😉